src/extension/debug-page/typeson-custom-types.ts

```diff
@@ -0,0 +1,15 @@
+export const CryptoKeyRegistry = {
+    test(x: any) {
+        return x instanceof CryptoKey
+    },
+    async replaceAsync(key: CryptoKey) {
+        return {
+            jwk: await crypto.subtle.exportKey('jwk', key),
```

Comment:
Plz add try catch

Reply:
if you can’t export, then Typeson should tell me
the exception not can ignore